wavefront_opentracing_sdk/span.py

```diff
@@ -92,6 +94,27 @@ def set_tag(self, key, value):
                         force_sampling)
         return self
 
+    def log_kv(self, key_values, timestamp=None):
+        """Add a log record to the span.
+
+        :param key_values: A dict of string keys and values of any type can be
+        transferred to string by str()
+        :type key_values: dict
+        :param timestamp: A unix timestamp per :meth:`time.time()`; current
+            time if ``None``
+        :type timestamp: float
+        :return: span itself
+        :rtype: WavefrontSpan
+        """
+        if key_values:
+            fields = {}
+            for key in key_values:
+                fields[key] = str(key_values.get(key))
+            with self.update_lock:
+                self.logs.append(
+                    SpanLog(timestamp=timestamp or time.time() * 1000000,
```

Comment:
Please be aware that `time.time()` is returning a `float`, and in case you expect timestamp to be `int` you may need to do something like `int(time.time() * 1E6)`